published/private/xdpioctl.h

```diff
@@ -41,6 +41,7 @@ typedef enum _XDP_OBJECT_TYPE {
 typedef struct _XDP_OPEN_PACKET {
     UINT16 MajorVersion;
     UINT16 MinorVersion;
+    UINT32 ApiVersion;
```

Comment:
Do we need to worry about IOCTL breaking changes (i.e. newer kernel, older user mode)?

Reply:
No, we agreed elsewhere the IOCTL interface is not version-stable.